src/capture/es_analyzer.cc

```diff
@@ -0,0 +1,635 @@
+#include "capture/es_analyzer.h"
```

Comment:
Please refactor `analyzer.{cc,h}` and `ac_analyzer.{cc,h}` first to reduce the duplicated code between `{es,ac}_analyzer`.